app/controllers/katello/api/v2/upstream_subscriptions_controller.rb

```diff
@@ -71,19 +73,6 @@ def update_params
       params.permit(pools: [:id, :quantity])[:pools].map(&:to_h)
     end
 
-    def upstream_pool_params
-      upstream_params = params.permit(:page, :per_page, :order, :sort_by, :quantities_only, :attachable, pool_ids: []).to_h
-
-      if params[:full_result]
-        upstream_params.delete(:per_page)
-        upstream_params.delete(:page)
-      elsif !params[:per_page]
-        upstream_params[:per_page] = Setting[:entries_per_page]
-      end
-
-      upstream_params
-    end
```

Comment:
does this still paginate requests when the pool_ids are *not* specified?  It looks like it got removed from both.

Reply:
oh i see what you mean, I mistakenly thought that was not in this api, oops.

fixing now